casper/src/main/resources/MakeMint.rho

```diff
@@ -31,18 +31,25 @@ in {
         } |
         contract internalMakePurse(balance, return) = {
           //balance must be a name that NonNegativeNumber contract is listening on
-          new thisPurse in {
+          new thisPurse, logStore in {
+            logStore!(Nil) |
             new letCh in { // let thisDecr = ...
               letCh!(bundle0{bundle0{*thisPurse}|*decr}) | for (thisDecr <- letCh) {
                 contract thisPurse(@=*thisDecr, @amount, return) = {
-                  new success in {
-                    balance!("sub", amount, *success) | for (@subOk <- success) {
-                      return!(subOk, *decr)
+                  for (logCh <<- logStore) {
+                    new success in {
+                      balance!("sub", amount, *success) | for (@subOk <- success) {
+                        if (Nil != *logCh) { logCh!(["*decr", amount, subOk]) } |
```

Comment:
Is there a meaning for `*` in the log message?

Reply:
That's the method being invoked.  It's not a string like "deposit", it's the process `*decr`.